pkgs/applications/video/kodi/unwrapped.nix

```diff
@@ -38,15 +38,15 @@ assert usbSupport -> !udevSupport; # libusb-compat-0_1 won't be used if udev is
 assert gbmSupport || waylandSupport || x11Support;
 
 let
-  kodiReleaseDate = "20210219";
-  kodiVersion = "19.0";
+  kodiReleaseDate = "20210508";
+  kodiVersion = "19.1";
   rel = "Matrix";
 
   kodi_src = fetchFromGitHub {
     owner  = "xbmc";
     repo   = "xbmc";
-    rev    = "${kodiVersion}-${rel}";
-    sha256 = "097dg6a7v4ia85jx1pmlpwzdpqcqxlrmniqd005q73zvgj67zc2p";
+    rev    = "v${kodiVersion}";
```

Comment:
I get a 404 error with the url this generates (https://github.com/xbmc/xbmc/archive/v19.1.tar.gz) . With the rev before this change, the url would be https://github.com/xbmc/xbmc/archive/19.1-Matrix.tar.gz. This url works just fine.

Reply:
Yeah it looks like they renamed it. I guess I'll have to file a PR to update that... unless you are able to tackle that?

Reply:
#123301